phoenix-core/src/main/java/org/apache/phoenix/calcite/rules/PhoenixConverterRules.java

```diff
@@ -899,9 +899,6 @@ public static boolean isConvertible(Aggregate input) {
         if (input.getGroupSets().size() > 1)
             return false;
         
-        if (input.containsDistinctCall())
```

Comment:
One minor nit, would you mind add check for other distinct functions? coz we don't support SUM DISTINCT yet.